templates/zerver/help/link-to-a-message-or-conversation.md

```diff
@@ -59,15 +59,15 @@ message](/help/quote-and-reply).
 
 1. Click **Copy link to message**.
 
-{tab|via-browser-address-bar}
+{tab|web}
```

Comment:
@gnprice Thanks for the feedback! That makes sense. I've made changes accordingly as shown in the image below, does that look good to you?

btw, I would suggest deleting "in the left sidebar" from the 1st step of the alternative way of doing it because topics can also be clicked in a stream narrow and, besides, I think users can learn that they can click on topics in the left sidebar from the first way of doing it. So perhaps "1. Click on a topic." should suffice.

<img width="533" alt="image" src="https://user-images.githubusercontent.com/2343554/171760257-d998e4da-0aea-43a3-84e8-9991561d7dc5.png">

Reply:
Mm, yeah, those are good points! Hmm, it's worth thinking about how we should handle alternative options for the web when we add mobile instructions. Ideally, we'd figure out a general paradigm that works well.

I don't recall having alternatives _within_ a tab elsewhere, though I'm not sure how to check.  These particular instructions are fairly short, but there are other pages where alternative variants are longer, and it might not be great to stack on them within a tab. 

For this particular case, I think we can avoid stacking alternatives by just naming the tabs differently. Something like "Desktop / Web", "Web (via browser's address bar)", "Mobile".

What do you guys think?

Reply:
@alya That's a really good point too! I think "Desktop / Web" is a great idea because it would tell the user that the same instructions work on both platforms, there wouldn't be stacked alternatives, and we wouldn't be duplicating instructions in two separate tabs.

As a general pattern, I think it would be ideal if the tabs always correspond to Zulip's supported platforms. So both  `Desktop/Web | Mobile` and `Desktop | Web | Mobile` would make sense.  They would be a useful UI to help users find what they are looking for depending on where they are using Zulip. Also, I think that using this pattern consistently across all user docs would make it easier to document features for everyone.

I like that "Web (via browser's address bar)" would be a very explicit tab name. But as opposed to the "Desktop" and "Mobile" tabs, it would be a lengthy name, and the phrase "browser's address bar" would show up again in the instructions. Also, I think it wouldn't really fit the pattern mentioned above if everyone were to agree on using that as a general pattern.

For this particular case, I would suggest `Desktop/Web | Mobile` to avoid stacking alternative steps and include the "via browser's address bar" as a Tip because I think it's short and adequate enough to be considered a tip.

I think the following would work well as it combines all the good points we have discussed so far :)
<img width="543" alt="image" src="https://user-images.githubusercontent.com/2343554/171881222-f87c3667-0265-4476-9d30-ad4f55cc23f9.png">

Reply:
The tip approach seems reasonable in this case, yeah. Maybe something like this for the wording:

> If using Zulip in the browser, you can also click on a topic name, and copy the URL from your browser's address bar.

(Note that you don't have to click on it from the topic bar.)

I don't think we've solved the general problem, because sometimes the alternative instructions are quite complex. If we put them into a separate tab, we'll need some way to name that tab so that it hints to the user what they'll get out of clicking on it. For instance, I'm thinking of the version of `/help/add-or-remove-users-from-a-stream#add-users-to-a-stream` in `main` (not yet deployed to `zulip.com`):

<img width="861" alt="Screen Shot 2022-06-03 at 2 09 35 PM" src="https://user-images.githubusercontent.com/2090066/171953299-0bab4b24-c61e-4b11-b345-90516b2ec1b9.png">

But it can potentially be a problem for another day.

Reply:
> The tip approach seems reasonable in this case, yeah. Maybe something like this for the wording:
> 
> > If using Zulip in the browser, you can also click on a topic name, and copy the URL from your browser's address bar.

Sounds good!

Thanks for showing the example you were thinking of. Yeah, I can see that's a more complex set of instructions. Hmm, maybe in this case it would be reasonable to create an "**Alternate methods**" heading and then create another tab switcher with the `Desktop/Web | Mobile` tabs to show alternatives? I would have to think more about it and test whether this idea would actually make sense though.

Reply:
That tip approach sounds good to me. I agree it won't scale to where the alternative instructions are longer, and I think it makes sense to leave that problem for when we're looking at an example of it.

Reply:
Ok, sounds like a plan! @drrosa could you please update the PR as discussed, and ping me when it's ready for review?